config/initializers/content_security_policy.rb

```diff
@@ -1,12 +1,16 @@
 Rails.application.config.content_security_policy do |p|
-  p.default_src :self, :https
+  p.default_src :self, :https, "http://localhost:*", "ws://localhost:*", "http://localhost:8080"
```

Comment:
doooooooo you want these in here?

Reply:
Thought I had made these dev only, good catch